include/CLI/Validators.hpp

```diff
@@ -240,6 +264,137 @@ struct Range : public Validator {
     template <typename T> explicit Range(T max) : Range(static_cast<T>(0), max) {}
 };
 
+/// Verify items are in a set
+class IsMember : public Validator {
+  public:
+    using filter_fn_t = std::function<std::string(std::string)>;
+
+    /// This allows in-place construction using an initializer list
+    template <typename... Args>
+    explicit IsMember(std::initializer_list<std::string> values, Args &&... args)
+        : IsMember(std::vector<std::string>(values), std::forward<Args>(args)...) {}
+
+    /// This checks to see if an item is in a set: pointer version. (Empty function)
+    template <typename T, enable_if_t<is_copyable_ptr<T>::value, detail::enabler> = detail::dummy>
+    explicit IsMember(T set)
```

Comment:
may want to consider std::move(set) for the pointer and list syntax for the inner IsMember call, otherwise you will probably get a few copies of Set made unnecessarily.  Still never sure if the compilers optimize away some of the copies in contexts like this.

Reply:
I think move is okay here.